src/DependencyInjection/Compiler/ApiProblemPass.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\DependencyInjection\Compiler;
+
+use Libero\ApiProblemBundle\EventListener\ApiProblemListener;
```

Comment:
this is a dependency on the other bundle's code so if this is loaded by default, it may force to put it in `require` rather than `suggests`?

Reply:
https://github.com/libero/content-api-bundle/blob/79f4624d7241d7affb74db56b5139066e87ea0d7/src/DependencyInjection/Compiler/ApiProblemPass.php#L18-L20 bails out if the bundle is not enabled, so it's safe.

Reply:
(`::class` doesn't trigger autoloading.)